workshops.md

```diff
@@ -15,3 +15,4 @@
 | 2025-01-28        | Netherlands eScience Center | Amsterdam, The Netherlands |          |
 | 2025-03-03        | CASTIEL2                    | Online                     | Part of a course `Multi-GPU AI Train-the-Trainer` |
 | 2025-05-06        | EuroCC National Competence Center Sweden | Online        |          |
+| 2025-10-27 to 29      |Helmholtz Zentrum Dresden Rossendorf, Germany | Online        |          |
```

Comment:
Just a minor detail: according to the Helmholtz Centre's own [About page](https://www.hzdr.de/db/Cms?pNid=139), the official spelling uses hyphenation in two places:

For the sake of standardized spelling, you might want to accept this suggestion:
```suggestion
| 2025-10-27 to 29      | Helmholtz-Zentrum Dresden-Rossendorf, Germany | Online        |          |
```
@joshigunjan Feel free to accept this or not, but I wanted to double-check before merging.